app/controllers/admin/roles_controller.rb

```diff
@@ -24,6 +24,8 @@ def create
 
   def edit
     @role = Role.find(params[:id])
+
+    render_design_system("edit", "legacy_edit", next_release: false)
   end
 
   def update
```

Comment:
we also need to update line 35 so when the save fails the new view is rerendered 

```
    render_design_system("edit", "legacy_edit", next_release: false)

```

Reply:
Cool, thanks. Makes sense ✅